sdk/storage/azure-storage-queue/src/main/java/com/azure/storage/queue/QueueConfiguration.java

```diff
@@ -9,5 +9,5 @@
 class QueueConfiguration {
     //TODO: Eventually remove these hardcoded strings with https://github.com/Azure/azure-sdk-for-java/issues/3141
     static final String NAME = "azure-storage-queue";
-    static final String VERSION = "12.0.0-preview.4";
+    static final String VERSION = "12.0.0-preview.5";
```

Comment:
Did you have to update this one manually? @JimSuplizio looks like we might need to add some support for the source code itself as well.

Reply:
@weshaggard Yeah, you're right. It would be trivial to update the script to deal with .java files, my concern is how long it'll take to process the entire tree. There are far less .md and pom files than there are java files. I'll do some experimentation.

Reply:
Yeah, this had to be done manually and @JimSuplizio is looking into automating it. We also have a github issue to remove these hardcoded version numbers from source code. If we do that, then updating the script won't be necessary.